build.yaml

```diff
@@ -554,7 +553,7 @@ job_type:
       clean_repo: true
 
     run_trial_for_cinder_storage_driver_on_Ubuntu_trusty:
-      on_nodes_with_labels: 'rackspace-ubuntu14-jenkins-slave-DFW-standard-4'
+      on_nodes_with_labels: 'rackspace-jenkins-slave-ubuntu14-standard-4-dfw'
```

Comment:
Not new so don't address now ftr I think that `ubuntu-14-` is a bad part of the label as there are multiple 2014 Ubuntus. Perhaps we should standardise on what the Flocker code uses which is `ubuntu-14.04`, `centos-7` etc.

All of these labels seem to have a different standard. Some include `jenkins-slave`, some say the region. I guess that there are good reasons for these but we should document them so that anyone can add a new image.